terms-of-service.md

```diff
@@ -43,28 +43,28 @@ including price changes.
 
 Whereby is only for people **16 years old and over**. By using
 Whereby, you affirm that you are over 16. If we learn that someone
-under 16 is using Whereby, we’ll terminate their account.
+under 16 is using Whereby, we’ll terminate their account. For use in educational settings or use of our Meetings API in your own service, [contact us](mailto:sales@whereby.com).
 
 ## 3. How you can use the Service
 
-Whereby is for legitimate individual use only (personal or business
-communication). Your use of Whereby requires that you have hardware,
+The Free version of Whereby is for legitimate individual use only (personal or business
+communication). Commercial use of Whereby as part of your own business offering requires one of our paid plans or the Meetings API. Your use of Whereby requires that you have hardware,
```

Comment:
Your editor (the github ui?) doesn't keep this to a 80-ish char line width. That's okay. But this sentence is very long, so might make sense to break up.

This is only about style, not that important. Especially for these files. :)